smart_contracts.rst

```diff
@@ -118,9 +118,9 @@ Balance Data Hash
 |  locksroot             | bytes32    | Root of merkle tree of all pending lock lockhashes                                    |
 +------------------------+------------+---------------------------------------------------------------------------------------+
 
-.. _balance-update-proof-message:
+.. _balance-proof-update-message:
```

Comment:
Just a note: looking at this `balance-proof-update-message` reference now, after https://github.com/raiden-network/spec/pull/131, maybe the `message` part is not correct, because this does not describe the Raiden client message. Consider something like `balance-proof-update-onchain`.

Reply:
Will do in this PR.